app/views/standard/addClosureMetadata.scala.html

```diff
@@ -53,7 +53,7 @@ <h2>@file.name</h2>
                   ),
                   fieldErrors
                 )
-                case DateField(fieldId, fieldName, fieldDescription, day, month, year, _, fieldErrors, _) => inputDate(
+                case DateField(fieldId, fieldName, fieldDescription, multiValue, day, month, year, _, fieldErrors, _) => inputDate(
```

Comment:
This parameter isn't being used, can you swap it out with `_` ? It's the same on line 40 and line 66 as well.

Reply:
Ah, yes, you are correct.